wave_vision/include/wave/vision/matcher/descriptor_matcher.hpp

```diff
@@ -89,7 +89,7 @@ class DescriptorMatcher {
       const cv::Mat &descriptors_2,
       const std::vector<cv::KeyPoint> &keypoints_1,
       const std::vector<cv::KeyPoint> &keypoints_2,
-      cv::InputArray mask = cv::noArray()) const = 0;
+      cv::InputArray mask) const = 0;
```

Comment:
@leokoppel this was actually the place where I mentioned CLion was complaining about overriding the default parameters of a virtual function, btw. It says "default arguments on virtual or override methods are prohibited." It's still yelling at me, actually.